tests/fv3_conf/rrfs_warm_run.IN

```diff
@@ -0,0 +1,25 @@
+rm -fr INPUT RESTART
+mkdir INPUT RESTART
+
+OPNREQ_TEST=${OPNREQ_TEST:-false}
+SUFFIX=${RT_SUFFIX}
+cp -r @[INPUTDATA_ROOT]/FV3_input_data_conus13km/INPUT/* INPUT/
+cp    @[INPUTDATA_ROOT]/FV3_input_data_conus13km/global* .
```

Comment:
To avoid duplications and take extra space, we have the fix data under @[INPUTDATA_ROOT]/FV3_fix, and these data are copied to run directory in run_test.sh under "# fix files" section.  Can you use those data instead of another copy the fix files under FV3_input_data_conus13km?

Reply:
I can't find global_soilmgldas.t126.384.190.grb in there.

Reply:
The resolution dependent file is located in FV3_input_data* (FV3_input_data if C96).

Reply:
That one file is in the input data directory, so that matches what the other tests have done. I'm using FV3_fix for everything else now.

Reply:
testing this now

Reply:
Fix files are no longer in the FV3_input_data_conus13km directory.